External/FEXCore/include/FEXCore/Utils/File.h

```diff
@@ -81,6 +82,10 @@ namespace FEXCore::File {
 #endif
       }
 
+      ssize_t Write(fextl::string const &Data) {
```

Comment:
Perhaps this should take a std::string_view, so that it can accept all sorts of string types instead of potentially converting to an allocating string

Reply:
switched to string_view